src/scenes/Question/scenes/Edit/components/Tips/Tips.jsx

```diff
@@ -1,16 +1,93 @@
 import React from 'react'
 
-import Card, { CardText } from 'components/Card'
+import { default as TipsComponent } from 'components/Tips'
 
 const Tips = props => (
-  <Card {...props}>
-    <CardText>
-      <h3 style={{ textAlign: 'center', color: 'var(--primary-color)' }}>
-        Tips to write great questions on the FAQ
-      </h3>
-      --Need help--
-    </CardText>
-  </Card>
+  <TipsComponent {...props} uid="tips_question">
+    <h3>Tips to write great questions on the FAQ</h3>
+    <div style={{ textAlign: 'center' }}>
+      <ul style={{ textAlign: 'left' }}>
+        <li>
+          Do some searches before you can be sure your question has not already
+          been asked.
+        </li>
+        <li>Be concise and precise in your wording.</li>
+        <li>
+          <span>
+            Privilege questions responding to the <b>how</b> rather than the{' '}
+            <b>why</b>.
+          </span>
+        </li>
+        <li>If the wording is not perfect, someone else can improve it.</li>
```

Comment:
I suggest "You don't have to strike for the perfect wording. Others can improve it."